.github/workflows/workflow.yml

```diff
@@ -11,29 +11,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: install dependencies 
+      - name: install dependencies
         run: yarn install --immutable
-      - name: run affected tests 
+      - name: generate workspaces
```

Comment:
As this command doesn't generate new workspaces I think "create autogenerated content" could be a better name